frontend/src/views/ArticleView.vue

```diff
@@ -1,7 +1,84 @@
 <script setup lang="ts">
-import ArticleList from "@/components/articles/ArticleList.vue";
+import Button from "primevue/button";
+import Card from "primevue/card";
+import Dropdown from "primevue/dropdown";
+import FloatLabel from "primevue/floatlabel";
+import InputText from "primevue/inputtext";
+import Textarea from "primevue/textarea";
+import { ref } from "vue";
+
+const articlesList = ref([{ id: 1 }, { id: 2 }, { id: 3 }]);
+
+let value = ref();
+let articleInfo = ref({
+  titel: null,
+  text: null,
+  unit: null,
+  price: null,
+});
```

Comment:
I think this makes more sense, no?

```suggestion
const articlesList = ref<Article[]>([
  {
    id: "1",
    kind: ArticleKind.heading,
    title: "title 1",
    description: "description 1",
  },
  {
    id: "2",
    kind: ArticleKind.item,
    title: "title 2",
    description: "description 2",
    unit: "unit 2",
    price: 2.2,
  },
  {
    id: "3",
    kind: ArticleKind.heading,
    title: "title 3",
    description: "description 3",
  },
]);
```